### PR TITLE
Vatrp 3012 submod fix

### PR DESCRIPTION
**Pull request by zackm0571**

### CODE REVIEW

**Comment by zackm0571** [comment]
+1 for Ruben's commits

**Comment by shareefalis** [comment]
revert the status bar change. They want the time to show anyways. 

EDIT: after talking with others. they want it, regard this comment

File: linphone-Info.plist [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string></string>
+	<string>1.0.17</string>
```
Comment (ianblenke) [inline linphone-Info.plist]: Please revert this.

**Comment by ianblenke** [comment]
After you revert that plist version number to be empty again, you have my +1

**Comment by shareefalis** [comment]
+1

**Pull request merged**